src/redux/categories/categories.js

```diff
@@ -0,0 +1,33 @@
+// Action Type
+// Reducer Function
+
+const checkStatus = (state = false, action) => {
+  switch (action.type) {
+    case 'STATUS':
+      return !state;
+    case 'ADD_CATEGORY':
+      return [...state, action.book];
+
+    case 'REMOVE_CATEGORY':
+      return state.filter((book) => book !== action.id);
+
+    default:
+      return state;
+  }
+};
+
+// Add A Book Category action
+
+export const addBook = (book) => ({
+  type: 'ADD_CATEGORY',
+  book,
+});
+
+// Remove a book Category action
+
+export const deleteBook = (id) => ({
+  type: 'REMOVE_CATEGORY',
+  id,
+});
+
+export default checkStatus;
```

Comment:
- [ ] Please note that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like the example :arrow_down:  . Please fix it in order to follow the requirement :smile: 

```
const STATUS = 'BOOKSTORE/books/STATUS';

export default (state = [], action) => {
  switch (action.type) {
    case STATUS:
      return ['Under construction'];
    default:
      return state;
  }
};

export const checkStatus = () => ({ type: STATUS });

```